site/content/docs/5.1/components/breadcrumb.md

```diff
@@ -54,8 +54,15 @@ $breadcrumb-divider: quote(">");
 
 It's also possible to use an **embedded SVG icon**. Apply it via our CSS custom property, or use the Sass variable.
 
+
+{{< callout info >}}
+### Using embedded SVG
+
+Inlining SVG as data URI requires to URL escape a few characters, most notably `<`, `>` and `#`. That's why the `$breadcrumb-divider` variable is passed through our [`escape-svg()` Sass function]({{< docsref "/customize/sass#escape-svg" >}}). When using the CSS custom property, you need to URL escape your SVG on your own. Read [Taylor Hunt's post "Optimizing SVGs in data URIs"](https://codepen.io/tigt/post/optimizing-svgs-in-data-uris) for detailed information on what to escape.
```

Comment:
In Sass we use this: https://github.com/twbs/bootstrap/blob/cd04fe015f9118930a86c678f034b5657878885a/scss/_functions.scss#L127

Maybe we should streamline the links?

Reply:
Indeed, we may add both links there, in our sources, and probably in the Sass functions page too ?

Is that what you have in mind?

Reply:
I just think we should use the same references is all :)